tools/src/test/python/dlpx/virtualization/_internal/commands/test_build.py

```diff
@@ -499,7 +500,26 @@ def test_schema_file_bad_permission(mock_generate_python,
                                         plugin_config_file, artifact_file,
                                         schema_file):
         # Make it so we can't read the file
-        os.chmod(schema_file, 0000)
+        if os.name == 'nt':
+	    pytest.skip('skipping this test on windows as os.chmod has issues removing permissions on file')
+	    # This code can be used to make the file unreadable on windows but
+	    # since it adds dependency on pypiwin32 for the sdk, i think its better
+	    # to skip this test instead of potentially destabilizing the sdk.
+	
+            # sd = win32security.GetFileSecurity(schema_file, win32security.DACL_SECURITY_INFORMATION)
+            # everyone, domain, type = win32security.LookupAccountName("", "Everyone")
+            # admins, domain, type = win32security.LookupAccountName("", "Administrators")
+            # user, domain, type = win32security.LookupAccountName("", win32api.GetUserName())
+
+            # dacl = win32security.ACL()
+            # dacl.AddAccessAllowedAce(win32security.ACL_REVISION, con.FAILED_ACCESS_ACE_FLAG, everyone)
+            # dacl.AddAccessAllowedAce(win32security.ACL_REVISION, con.FAILED_ACCESS_ACE_FLAG, user)
+            # dacl.AddAccessAllowedAce(win32security.ACL_REVISION, con.FAILED_ACCESS_ACE_FLAG, admins)
+
+            # sd.SetSecurityDescriptorDacl(1, dacl, 0)
+            # win32security.SetFileSecurity(schema_file, win32security.DACL_SECURITY_INFORMATION, sd)
```

Comment:
did you mean to leave these as comments? maybe for future reference? seems like if we're just gonna skip we might want to remove these comments?